src/Microsoft.AspNet.Mvc.Core/Controller.cs

```diff
@@ -357,10 +358,22 @@ public virtual ContentResult Content(string content, string contentType)
         [NonAction]
         public virtual ContentResult Content(string content, string contentType, Encoding contentEncoding)
         {
+            return Content(content, new MediaTypeHeaderValue(contentType) { Encoding = contentEncoding });
+        }
+
+        /// <summary>
+        /// Creates a <see cref="ContentResult"/> object by specifying a <paramref name="content"/>
+        /// string and a <paramref name="contentType"/>.
+        /// </summary>
+        /// <param name="content">The content to write to the response.</param>
+        /// <param name="contentType">The content type (MIME type).</param>
+        /// <returns>The created <see cref="ContentResult"/> object for the response.</returns>
+        [NonAction]
+        public virtual ContentResult Content(string content, MediaTypeHeaderValue contentType)
```

Comment:
There are lot of helper methods which accept content type as string, should i have corresponding helpers for MediaTypeHeaderValue too? wanted to check first as there are lot of methods to add.